static/app/utils/theme/theme.tsx

```diff
@@ -1036,6 +1036,7 @@ const darkAliases = generateThemeAliases(darkColors);
 export const lightTheme = {
   isChonk: false,
   ...commonTheme,
+  ...formTheme,
```

Comment:
I moved these out of the `commonTheme` so that we also have to apply them to `chonk` on type level, but maybe a normal override would’ve been fine, too